xclim/ensembles/_base.py

```diff
@@ -207,6 +217,7 @@ def ensemble_percentiles(
     min_members: int | None = 1,
     weights: xr.DataArray | None = None,
     split: bool = True,
+    method: str = "linear",
```

Comment:
you can also use `Literal` to force the user to select one of the supported methods

Reply:
Wow, I had no idea! Thanks for the suggestion!

Reference: https://typing.readthedocs.io/en/latest/spec/literal.html